.github/workflows/static-analysis.yaml

```diff
@@ -13,7 +13,7 @@ jobs:
       - name: golangci-lint
         uses: golangci/golangci-lint-action@aaa42aa0628b4ae2578232a66b541047968fac86 # v6.1.0
         with:
-          version: v1.59.1
+          version: v1.60.1
```

Comment:
I spotted this also in bbolt. Do we want to lock a version here? I think the only way to bump the version would be manually. Would it be better to set it to `latest` (refer to their [documentation](https://github.com/golangci/golangci-lint-action?tab=readme-ov-file#version)).

@jmhbnz, WDYT?

Reply:
I think maybe we would like to follow the dependency pinning everywhere, though using the `latest` tag would be nicer for the linter since the update can come right away.

Reply:
Yes, however, dependabot won't update this. I think it may also not be that relevant when we migrate these jobs to Prow.